### PR TITLE
Document POST /v1/appusers

### PR DESCRIPTION
**Pull request by alavers**
@mspensieri @lemieux @dannytranlx @jpjoyal @Mario54

docs for https://github.com/smooch/smooch/pull/556

### CODE REVIEW

File: source/rest/index.md [hunk 1]
```diff
@@ -554,6 +554,56 @@ Trigger an event for a given app user. Some Smooch whispers are triggered on dis
...
+| **signedUpAt**<br/>*optional* | The date at which the user signed up. Must be ISO 8601 time format (`YYYY-MM-DDThh:mm:ss.sssZ`) |
+| **properties**<br/>*optional* | A flat JSON object containing custom defined user properties. |
+
+In the vast majority of cases app users will be created from the device or browser using the [init API](#init-beta). In some cases however it might be necessary to pre-create an app user object before that user runs your app for the first time. This API facilitates this scenario. A `userId` must be specified so that a future `init` call made from a device can use the same `userId` to link the device to the pre-created app user.
+
+Suppose for example you begin a conversation with an end user `bob@example.com` over email and you wish to transfer this conversation history over into Smooch once that user logs in to your app. To facilitate this, you can call `POST /v1/appusers` to pre-create a Smooch identity with `userId` `bob@example.com`, to which you can import that existing conversation history. After Bob signs in to your app and your app calls `init` with the same `uesrId`, they will see their conversation history.
```
Comment (mspensieri) [inline source/rest/index.md]: typo `uesrId`

**Comment by mspensieri** [comment]
:+1:

File: source/rest/index.md (hunk 1 above)
Comment (lemieux) [inline source/rest/index.md]: how would one transfer the history to the new conversation?

File: source/rest/index.md [hunk 2]
```diff
@@ -554,6 +554,56 @@ Trigger an event for a given app user. Some Smooch whispers are triggered on dis
...
+
+<aside class="notice">
+Unlike the other App User APIs in this section, this endpoint is not intended to be called form an end user device or from a browser. It requires a `jwt` credential with `app` level scope.
+</aside>
+
 # Conversations <beta/>
```
Comment (lemieux) [inline source/rest/index.md]: not part of your changes .. but is it still beta?

File: source/rest/index.md (hunk 1 above)
Comment (alavers) [inline source/rest/index.md]: with the app user conversation API :)

File: source/rest/index.md (hunk 2 above)
Comment (alavers) [inline source/rest/index.md]: Yes, beta tag is is on the App User section header

**Comment by lemieux** [comment]
:+1: I guess I'll update smooch-core to add this endpoint

**Pull request merged**